owncloudDomain/src/main/java/com/owncloud/android/domain/files/model/OCFile.kt

```diff
@@ -57,6 +57,7 @@ data class OCFile(
     var sharedWithSharee: Boolean? = false,
     var sharedByLink: Boolean = false,
     val spaceId: String? = null,
+    val previousUri: String? = ""
```

Comment:
Add a coma. Why the name is "previousUri"? I would put something like "thumbnailUri"

Reply:
Done